install_config/cluster_metrics.adoc

```diff
@@ -39,7 +39,7 @@ Hawkular Metrics.
 
 The storage volume metrics available on the kubelet are not available through
 the `/stats` endpoint, but are available through the `/metrics` endpoint. See
-{product-title} via Prometheus for detailed information.
+xref:../install_config/prometheus_cluster_monitoring.adoc[Prometheus Monitoring] for detailed information.
```

Comment:
xref:../install_config/prometheus_cluster_monitoring.adoc#prometheus-cluster-monitoring[Prometheus Monitoring]

This _should_ fix the build failure.